### PR TITLE
add: Go programming language Persian book

### PR DESCRIPTION
**Pull request by Ja7ad**
## What does this PR do?
Add Go programming language book for Persian gopher

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-fa_IR.md [hunk 1]
```diff
@@ -35,6 +36,9 @@
 
 * [یادگیری پیکربندی با CSS](http://fa.learnlayout.com)
 
+### Go
+
+* [کتاب آزاد آموزش زبان گو](https://book.gofarsi.ir) - [مخزن کتاب](https://github.com/GoFarsi/book)
```
Comment (eshellman) [inline books/free-programming-books-fa_IR.md:41]: just one link please. linter wants 2 spaces after a section.

File: books/free-programming-books-fa_IR.md [hunk 2]
```diff
@@ -35,6 +36,9 @@
 
 * [یادگیری پیکربندی با CSS](http://fa.learnlayout.com)
 
+### Go
```
Comment (shrn01) [inline books/free-programming-books-fa_IR.md:39]:
```suggestion

### Go
```
2 spaces after each section

File: books/free-programming-books-fa_IR.md [hunk 3]
```diff
@@ -35,6 +36,9 @@
...
+
+* [کتاب آزاد آموزش زبان گو](https://book.gofarsi.ir)
```
Comment (shrn01) [inline books/free-programming-books-fa_IR.md:41]:
```suggestion
* [کتاب آزاد آموزش زبان گو](https://book.gofarsi.ir)

```
2 spaces after each section

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

**Comment by Ja7ad** [comment]
> The list in the index must be in alphabetical order. So "Software Engineering" comes after "R". Perhaps it's better to leave the Category headings in Farsi for the PR where you add the Go book.

All things passed, thank you.

**Comment by eshellman** [comment]
Book is fine, but the headings in Farsi have been there for 7+ years please explain why they should be changed to English. Perhaps @badrnezhad or other Farsi contributors can comment?

**Comment by Ja7ad** [comment]
>

**Comment by Ja7ad** [comment]
> Book is fine, but the headings in Farsi have been there for 7+ years please explain why they should be changed to English. Perhaps @badrnezhad or other Farsi contributors can comment?

I think sections better write in English. This name is known as English and we speak in Persian about English name.

**Pull request closed without merging**